src/pkg/transform/image.go

```diff
@@ -63,39 +63,52 @@ func ImageTransformHostWithoutChecksum(targetHost, srcReference string) (string,
 
 // ParseImageRef parses a source reference into an Image struct
 func ParseImageRef(srcReference string) (out Image, err error) {
-	ref, err := reference.ParseAnyReference(srcReference)
-	if err != nil {
-		return out, err
-	}
-
-	// Parse the reference into its components
-	if named, ok := ref.(reference.Named); ok {
-		out.Name = named.Name()
-		out.Path = reference.Path(named)
-		out.Host = reference.Domain(named)
-		out.Reference = ref.String()
+	if IsTarball(srcReference) {
+		out.Host = "docker.io"
+		out.Name = strings.TrimSuffix(srcReference, ".tar")
+		out.Path = fmt.Sprintf("%s/%s/%s", out.Host, "library", out.Name)
+		out.Reference = srcReference
```

Comment:
shall make the changes 
I was able to deploy the package though created from this logic 🤔 not sure how

Reply:
![image](https://github.com/defenseunicorns/zarf/assets/3977569/72a955ff-d1e7-4c31-8775-3576d5292d8e)
![image](https://github.com/defenseunicorns/zarf/assets/3977569/39015237-e69a-452d-a22e-2ecf1965ad13)
![image](https://github.com/defenseunicorns/zarf/assets/3977569/166f7041-6045-4ad1-86d5-ca04b2b98ad0)
![image](https://github.com/defenseunicorns/zarf/assets/3977569/3a6ec37c-794b-4fd3-ae7f-3da7e43587d4)

the reason is that the cluster caches the image and this same image is used in earlier tests (hence the passes in CI as well).  On a fresh cluster / registry it will fail with an ImagePullBackoff.

Note `127.0.0.1:31999/defenseunicorns/zarf-game:multi-tile-dark-zarf-805192805` differs from `127.0.0.1:31999/docker.io/library/zarf-game:latest-zarf-27383200`